Makefile

```diff
@@ -55,12 +58,17 @@ else
 	cargo build --bin piglet --features "fake_hw"
 endif
 
-.PHONY: cross–build
+.PHONY: cross-build cross-build-pi3b
```

Comment:
Is this .PHONY in the right place?
Should be just before the target you added and have the same target as a dependency